python/pyspark/testing/sqlutils.py

```diff
@@ -243,6 +243,13 @@ def function(self, *functions):
             for f in functions:
                 self.spark.sql("DROP FUNCTION IF EXISTS %s" % f)
 
+    @staticmethod
```

Comment:
I moved get_values and assert_close and combined them as a static method under SQLTestUtils as I don't see any function on the top-level statement. 
Please let me know if it should be on the top level rather than a method.

Reply:
Yeah looks good.